backend/products-ms/src/common/utils/api-response.util.ts

```
@@ -0,0 +1,35 @@
+        statusCode: HttpStatus = HttpStatus.BAD_REQUEST,
+    ): ApiResponse<void> {
+        return {
+            success: false,
+            statusCode,
+            message,
+            path,
+            timestamp: new Date().toISOString(),
+            data: undefined,
+        };
+    }
+} 
```

Biome (1.9.4)

[error] Avoid classes that contain only static members.

Prefer using simple functions instead of classes with only static members.

(lint/complexity/noStaticOnlyClass)